src/test/FreeC/IR/InliningTests.hs

```diff
@@ -0,0 +1,90 @@
+-- | This moduule contains tests for FreeC.IR.Inlining
```

Comment:
Use Haddock markup if your comments contain code or identifiers and end sentences with a period.

```suggestion
-- | This module contains tests for "FreeC.IR.Inlining".
```